src/components/Link/tests/Link.test.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { mount, shallow } from 'enzyme';
+import Link from '../Link';
+
+describe('onClick()', () => {
+  it('is called when the Link is clicked', () => {
+      const spy = jest.fn();
+      shallow(<Link onClick={spy}>Test</Link>).simulate('click');
+      expect(spy).toHaveBeenCalled();
+  });
+});
+
+describe('external', () => {
+  it('should verify the external is false', () => {
+      const linkWrapper = mount(<Link external={false}/>);
```

Comment:
`<Link external={false}/>` should have a space after } and before / `<Link external={false} />`